src/setup/type-specs/sample-type-spec.json

```diff
@@ -1,34 +1,20 @@
 {
   "types": {
-    "User": {
+    "user": {
```

Comment:
Let's put these back how to capitalized. Even though we fix it, I think it makes more sense to have the sample reflect what the typespec should look like